apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2.Tests/QueryOptionsTest.cs

```diff
@@ -23,6 +23,11 @@ public class QueryOptionsTest
         [Fact]
         public void PropertiesSetOnRequest()
         {
+            ConnectionProperty cp = new ConnectionProperty
+            {
+                Key = "session_id",
```

Comment:
This should probably be a constant somewhere or transparent to the user.

I find usability of this a little bit clunky. In the past we've used these option classes to help with usability, instead of surfacing API fields verbatim, we've created more specific properties that we then translate to the API fields. A good example of these are enums that we've created instead of exposing string constants.

I think something like this could be done for this feature. Maybe alternated SessionId and CreateSession properties, or similar, instead of having the user deal with the key-value collection, etc.

@jskeet What do you think?

Reply:
I will create a doc and add some options over there lets discuss there.